package.json

```diff
@@ -27,81 +27,77 @@
     "e2e:ci:server:down": "docker compose down",
     "e2e:ci": "docker compose -f e2e/docker/docker-compose.playwright.yaml up --build"
   },
-  "packageManager": "npm@10.5.0",
+  "packageManager": "npm@10.9.2",
   "engines": {
     "node": ">=22"
   },
   "dependencies": {
     "@bsull/augurs": "^0.9.0",
-    "@emotion/css": "11.10.6",
-    "@grafana/data": "^11.6.0",
-    "@grafana/faro-web-sdk": "^1.14.1",
+    "@emotion/css": "11.13.5",
+    "@grafana/data": "^11.6.1",
+    "@grafana/faro-web-sdk": "^1.17.2",
     "@grafana/lezer-logql": "^0.2.7",
-    "@grafana/prometheus": "^11.6.0-223871",
-    "@grafana/runtime": "^11.6.0",
+    "@grafana/prometheus": "^12.1.0-242149",
+    "@grafana/runtime": "^11.6.1",
     "@grafana/scenes": "^6.10.2",
-    "@grafana/schema": "^11.6.0",
-    "@grafana/ui": "^11.6.0",
+    "@grafana/schema": "^11.6.1",
+    "@grafana/ui": "^11.6.1",
     "@lezer/common": "^1.2.3",
     "p-limit": "^6.2.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
     "react-router-dom": "^6.30.0",
-    "rxjs": "7.8.1",
+    "rxjs": "7.8.2",
```

Comment:
It's unfortunate that upgrading `rxjs` from `7.8.1` to `7.8.2` is leading to `as any` and `as unknown`s that weren't previously there. It appears this is due to `@grafana/data` version `11.6.1` having a dependency on `7.8.1`.

My suggestion is to first try upgrading the `@grafana/<thing>@11.6.1` packages to `@latest`, then see if we can continue to use the generics properly.